DOCSBUILD.md

```diff
@@ -0,0 +1,347 @@
+# Documentation Build Process
```

Comment:
It might be a good idea to merge this document with the documentation we have on [ReadTheDocs](https://data-object-service.readthedocs.io/en/latest/contributors.html)